crates/bindings-csharp/Codegen.Tests/fixtures/server/snapshots/Module#PublicTable.verified.cs

```diff
@@ -188,165 +188,290 @@ SpacetimeDB.BSATN.ITypeRegistrar registrar
-                    new(nameof(IdentityField), BSATN.IdentityField.GetAlgebraicType(registrar)),
-                    new(nameof(AddressField), BSATN.AddressField.GetAlgebraicType(registrar)),
-                    new(
-                        nameof(CustomStructField),
-                        BSATN.CustomStructField.GetAlgebraicType(registrar)
-                    ),
-                    new(
-                        nameof(CustomClassField),
-                        BSATN.CustomClassField.GetAlgebraicType(registrar)
-                    ),
-                    new(nameof(CustomEnumField), BSATN.CustomEnumField.GetAlgebraicType(registrar)),
-                    new(
-                        nameof(CustomTaggedEnumField),
-                        BSATN.CustomTaggedEnumField.GetAlgebraicType(registrar)
-                    ),
-                    new(nameof(ListField), BSATN.ListField.GetAlgebraicType(registrar)),
-                    new(nameof(DictionaryField), BSATN.DictionaryField.GetAlgebraicType(registrar)),
-                    new(
-                        nameof(NullableValueField),
-                        BSATN.NullableValueField.GetAlgebraicType(registrar)
-                    ),
-                    new(
-                        nameof(NullableReferenceField),
-                        BSATN.NullableReferenceField.GetAlgebraicType(registrar)
-                    ),
-                    new(
-                        nameof(ComplexNestedField),
-                        BSATN.ComplexNestedField.GetAlgebraicType(registrar)
-                    )
-                ],
-                Indexes: [],
-                Constraints:
-                [
-                    new(
-                        nameof(PublicTable),
-                        0,
-                        nameof(Id),
-                        SpacetimeDB.ColumnAttrs.PrimaryKeyIdentity
-                    )
-                ],
-                Sequences: [],
-                // "system" | "user"
-                TableType: "user",
-                // "public" | "private"
-                TableAccess: "private",
-                Scheduled: null
+                new(
+                    TableName: nameof(PublicTable),
+                    Columns:
+                    [
+                        new(nameof(Id), global::PublicTable.BSATN.Id.GetAlgebraicType(registrar)),
+                        new(
+                            nameof(ByteField),
+                            global::PublicTable.BSATN.ByteField.GetAlgebraicType(registrar)
+                        ),
+                        new(
+                            nameof(UshortField),
+                            global::PublicTable.BSATN.UshortField.GetAlgebraicType(registrar)
+                        ),
+                        new(
+                            nameof(UintField),
+                            global::PublicTable.BSATN.UintField.GetAlgebraicType(registrar)
```

Comment:
Are all these changes necessary? I think in this case `BSATN` is still unambiguous even in face of multiple table views?

Reply:
Good catch, yeah this one doesn't need it.